src/components/atoms/ActionButtonV2/ActionButtonV2.js

```diff
@@ -0,0 +1,60 @@
+import styles from '!!raw-loader!./ActionButtonV2.css';
+import varStyles from '!!raw-loader!../../../shared/variables.css';
+import bootstrapStyles from '!!raw-loader!../../../shared/themed-bootstrap.css';
+
+const template = document.createElement('template');
+
+template.innerHTML = `
+<div class="action-button-container">
+  <a class="btn" role="button" href="">
+    <div class="abutton">
+      <div class="top-icon">
+        <cod-icon data-icon="" data-size="x-large">
+        </cod-icon>
+      </div>
+      <div class="abutton-body">  
+        <slot></slot>
+      <slot name="primary"></slot>
+      <slot name="secondary"></slot> 
```

Comment:
Glad you got it working @sreidthomas 👌 

We shouldn't need to use named slots here though. From [MDN](https://developer.mozilla.org/en-US/docs/Web/API/Web_components/Using_templates_and_slots#adding_flexibility_with_slots):

> Note: An unnamed [<slot>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/slot) will be filled with all of the custom element's top-level child nodes that do not have the [slot](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes#slot) attribute. This includes text nodes.

So we should be able to remove the extra two slots with `name=primary` and `name=secondary` and just leave the unnamed slot, then remove the slot attribute from the storybook elements. They should all get placed in the unnamed slot by default.

Reply:
Also, I'll send the notes on the icon implementation this afternoon.